src/Tooltip/Tooltip.js

```diff
@@ -281,6 +284,8 @@ module.exports = kind(
 		if (this.contentUpperCase !== null) this.uppercase = this.contentUpperCase;
 
 		this.contentChanged();
+
+		this.$.point.setAttribute('viewBox', '0 0 3 5');
```

Comment:
can we put this inside the components block instead? 
e.g. `{name: 'point', kind: Control,  ... , attributes: {viewBox: '0 0 3 5'}`